src/v/pandaproxy/reply.h

```diff
@@ -76,50 +77,71 @@ errored_body(std::error_condition ec, ss::sstring msg) {
     return rep;
 }
 
+inline std::unique_ptr<ss::httpd::reply>
+errored_body(std::error_condition ec, ss::sstring msg, error_probe& eprobe) {
+    auto rep = errored_body(ec, std::move(msg));
+    eprobe.increment_error_count();
```

Comment:
Not all errors are equal. Should the status be checked for >=400, or >=500? Should we have a label for `3xx`, `4xx`, `5xx`? There are options, here.

Reply:
That's a good point. I think adding a label for different error groups makes sense here.

Reply:
I shall make the change.

Reply:
done